pkgs/applications/networking/cluster/tilt/default.nix

```diff
@@ -1,29 +1,27 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ lib, buildGoPackage, fetchFromGitHub }:
 
 buildGoPackage rec {
-  pname   = "tilt";
-  name    = "${pname}-${version}";
+  pname = "tilt";
   /* Do not use "dev" as a version. If you do, Tilt will consider itself
      running in development environment and try to serve assets from the
      source tree, which is not there once build completes.  */
-  version = "0.8.8";
-  rev = "344dc1dc61ffe2c29606b105cea0df79fb5897f5";
+  version = "0.9.2";
 
   src = fetchFromGitHub {
     owner  = "windmilleng";
     repo   = "tilt";
-    rev    = "${rev}";
-    sha256 = "13yda6m2d92mmc9w4k8ngdxmpqcqf86bkrvcpmpaby848ls1yx8g";
+    rev    = "v${version}";
+    sha256 = "1kwama90bxmnylmj6hipnx5raiqq42dq31lb66xyd89fzbkdnzk4";
   };
 
   goPackagePath = "github.com/windmilleng/tilt";
   subPackages = [ "cmd/tilt" ];
 
-  buildFlagsArray = ("-ldflags=-X main.version=${version} -X main.date=2019-06-03");
+  buildFlagsArray = ("-ldflags=-X main.version=${version} -X main.date=2019-07-10");
 
-  meta = with stdenv.lib; {
+  meta = with lib; {
     description = "Local development tool to manage your developer instance when your team deploys to Kubernetes in production";
-    homepage = https://tilt.dev/;
+    homepage = "https://tilt.dev/";
```

Comment:
Why?

Reply:
This is being discussed as a new policy (NixOS/rfcs#45).
I think that it would help in the future.